### DISCUSSION TITLE
Gradio update

### DISCUSSION THREAD

**Discussion by SGThirkell** [lllyasviel/Fooocus#2889] [Q&A] [answered]
Hi, just got a notification in the window during Fooocus start, that Gradio has an update. I'm quite new to repo stuff. Should I update as I would in any other PC program I have, or not? If so, how is this done please?

**Comment by mashb1t** [comment, marked as answer]
@SGThirkell updating gradio is not done easily, which is also why the feature is in draft state since November and not being worked on, see https://github.com/lllyasviel/Fooocus/pull/961. If you have the capacity feel free to update locally, adjust all UI elements and overrides, adjust JavaScript event listeners and functions, modify css layout and contribute in a new PR.

**Reply by SGThirkell** [reply]
Thank you for your reply. I have no knowledge to do anything like this and so, will not attempt it.
